RestImageResize/web.config.transform

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <appSettings>
     <add key="RestImageResize.DefautTransform" value="downFit"/>
+    <add key="RestImageResize.PrivateKeys" value="" />
```

Comment:
maybe it makes sense to add a comment like "put semicolon separated private keys" here to start securing your image requests"